Source/WebCore/loader/CookieJar.cpp

```diff
@@ -123,10 +123,10 @@ void CookieJar::setCookies(Document& document, const URL& url, const String& coo
 {
     auto pageID = document.pageID();
     std::optional<FrameIdentifier> frameID;
-    if (auto* frame = document.frame())
+    if (RefPtr frame = document.frame())
         frameID = frame->loader().frameID();
```

Comment:
Couldn't these just be `frameID = frame->frameID();`? Then we wouldn't need the smart pointer.